localstack/aws/chain.py

```diff
@@ -61,15 +66,18 @@ def __init__(
         self,
         request_handlers: List[Handler] = None,
         response_handlers: List[Handler] = None,
+        finalizers: List[Handler] = None,
         exception_handlers: List[ExceptionHandler] = None,
     ) -> None:
         super().__init__()
         self.request_handlers = request_handlers or list()
         self.response_handlers = response_handlers or list()
         self.exception_handlers = exception_handlers or list()
+        self.finalizers = finalizers or list()
 
         self.stopped = False
         self.terminated = False
+        self.finalized = False
```

Comment:
q: when is this set to `True`?  or how can you set it to `True`?

Reply:
A handler could set it. It's just a way to control things for tests oder other systems that may not want/need familzers

Reply:
Ah right yes thanks! The `Handler` has access to the `HandlerChain`. Thanks for answering 😄